arangod/Agency/Agent.cpp

```diff
@@ -816,24 +816,31 @@ void Agent::sendEmptyAppendEntriesRPC(std::string const& followerId) {
   auto ac = std::make_shared<AgentCallback>(this, followerId, 0, 0);
 
   network::RequestOptions reqOpts;
+  reqOpts.skipScheduler = true;
   reqOpts.timeout = network::Timeout(3 * _config.minPing() * _config.timeoutMult());
   reqOpts.param("term", std::to_string(_constituent.term())).param("leaderId", id())
          .param("prevLogIndex", "0")
          .param("prevLogTerm", "0")
          .param("leaderCommit", std::to_string(commitIndex))
          .param("senderTimeStamp", std::to_string(std::llround(steadyClockToDouble() * 1000)));
 
+  double now = TRI_microtime();
   network::sendRequest(cp, _config.poolAt(followerId), fuerte::RestVerb::Post, "/_api/agency_priv/appendEntries",
                        std::move(buffer), reqOpts).thenValue([=](network::Response r) {
     ac->operator()(r);
   });
+  double diff = TRI_microtime() - now;
+  if (diff > 0.01) {
+    LOG_TOPIC("cfb7c", DEBUG, Logger::AGENCY)
+        << "Calling network::sendRequest took more than 1/100 of a second: " << diff;
+  }
```

Comment:
If something takes longer than 10 ms, that is at least not completely unexpected on a loaded server.

Reply:
This is debug output. Please leave as is as we have made this changes recently for being able to understand customer issues.